tests/sentry/grouping/snapshots/test_variants/test_event_hash_variant/mobile@2021_02_12/stacktrace_group_modules_1.pysnap

```diff
@@ -0,0 +1,36 @@
+---
+created: '2021-02-23T10:49:25.824614Z'
+creator: sentry
+source: tests/sentry/grouping/test_variants.py
+---
+app-depth-max:
+  hash: null
+  component:
+    app-depth-max (stacktrace of system takes precedence)
+      stacktrace
+        frame (non app frame)
+        frame (non app frame)
+        frame (non app frame)
+        frame (non app frame)
+        frame (non app frame)
+        frame (non app frame)
+          function*
+            "do_something"
+        frame (non app frame)
+        frame (non app frame)
+--------------------------------------------------------------------------
+system:
+  hash: "d66c268854f0e9a3bb23fad03cb69796"
+  component:
+    system*
+      stacktrace*
+        frame
+        frame (ignored due to recursion)
+        frame (ignored due to recursion)
+        frame
```

Comment:
Aren't these frames missing the package component? Also, they should be deduplicated.

Reply:
@Swatinem Deduplication still needs to be addressed, I think?